db/fixtures/ae_datastore/ManageIQ/ConfigurationManagement/AnsibleTower/Operations/StateMachines/Job.class/__methods__/launch_ansible_job.rb

```diff
@@ -0,0 +1,91 @@
+#
+# Description: Launch a Ansible Job Template and save the job id
+#              in the state variables so we can use it when we
+#              wait for the job to finish.
+#
+
+class LaunchAnsibleJob
+  ANSIBLE_VAR_REGEX = Regexp.new(/(.*)=(.*$)/)
+  SCRIPT_CLASS = 'ConfigurationScript'.freeze
+  JOB_CLASS = 'ManageIQ_Providers_AnsibleTower_ConfigurationManager_Job'.freeze
+
+  def initialize(handle)
+    @handle = handle
+  end
+
+  def main
+    target
+    job_template
+    run
+  end
+
+  private
+
+  def target
+    vm = @handle.root['vm'] || vm_from_request
+    @server_ip = vm.name if vm
```

Comment:
If you rework the `run` method then you no longer need to set the `@server_ip`.  If you do end up needing that variable it should be rename to represent the data it now contains.